southworkscom/botframework-solutions
# [TypeScript][Generator] Update the replacement of template's files

## Purpose
_What is the context of this pull request? Why is it being done?_

Update the replacement of templates files in order to have the same behavior between different Operating Systems.
Fix 2059

## Changes
_Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)_

Add into `generator-botbuilder` a logic to remove the files starting with character "_" after generating the `VA`/`Skill`.

![image](https://user-images.githubusercontent.com/40918752/62898810-c69e8900-bd2c-11e9-9a86-3cdb4e72905e.png)


## Testing Steps
1. Go to `.\templates\Virtual-Assistant-Template\typescript\generator-botbuilder-assistant`
1. Open a terminal in that location and execute `npm link` to link the package to the global `node_modules`  folder.
1. Open a terminal in a new folder outside the repository and execute `yo botbuilder-assistant` or `yo botbuilder-assistant:skill`.
1. Verify that everything is ok

## Feature Plan
_Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues._
\-

## Checklist
